collector/bonding_linux.go

```diff
@@ -31,6 +32,7 @@ type bondingCollector struct {
 
 func init() {
 	Factories["bonding"] = NewBondingCollector
+	CollectorsEnabledState["bonding"] = flag.Bool("collectors.bonding.enabled", false, "enables bonding-collector")
```

Comment:
Not saying it's wrong but why using a global map? I imagined just having a (global) flag per collector to enable it, which would toggle adding the collector instance via the Factories map.